src/ResourceManager/Sql/Commands.Sql/Auditing/Services/SqlAuditAdapter.cs

```diff
@@ -485,7 +485,7 @@ private BlobAuditingCreateOrUpdateParameters PolicizeBlobAuditingModel(BaseBlobA
 
         private static IList<string> ExtractAuditActionsAndGroups(BaseBlobAuditingPolicyModel model)
         {
-            var dbPolicyModel = model as DatabaseBlobAuditingPolicyModel;
+            var dbPolicyModel = model as DatabaseBlobAuditingSettingsModel;
             var actionsAndGroups = new List<string>();
             if (dbPolicyModel != null)
```

Comment:
do we want to add a trace for            
  if (dbPolicyModel == null)

Reply:
It's expected for server level